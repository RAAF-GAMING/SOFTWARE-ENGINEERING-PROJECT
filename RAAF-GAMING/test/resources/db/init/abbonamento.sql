
CREATE TABLE IF NOT EXISTS abbonamento (
  `codice` char(11) NOT NULL,
  `prodotto` int  NOT NULL,
  `durata_abbonamento` smallint NOT NULL DEFAULT '1',
  PRIMARY KEY (`codice`,`prodotto`)
);