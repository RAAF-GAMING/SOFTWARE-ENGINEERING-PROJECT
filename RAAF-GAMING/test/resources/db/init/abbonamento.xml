<?xml version="1.0" encoding="UTF-8"?>
<dataset>
    <abbonamento codice='asd34gf56d1' prodotto='5' durata_abbonamento='2'/>
    <abbonamento codice='qwerty65jl1' prodotto='7' durata_abbonamento='6'/>
</dataset>