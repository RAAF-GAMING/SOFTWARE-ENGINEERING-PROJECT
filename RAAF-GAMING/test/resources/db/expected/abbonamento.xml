<?xml version="1.0" encoding="UTF-8"?>
<dataset>
    <abbonamento codice='asd34gf56d1' prodotto='5' durata_abbonamento='2'/>
    <abbonamento codice='qwerty65jl1' prodotto='7' durata_abbonamento='6'/>
    <abbonamento codice='abcdef122f5' prodotto='8' durata_abbonamento='9'/>
</dataset>